spec/book_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rspec'
+require './book'
+require './rental'
+require './person'
+
+describe Book do
+  before(:each) do
+    @book = Book.new('if tomorrow comes', 'sidney sheldon')
+  end
+
+  context 'when creating a new book' do
+    it 'should create a new book' do
+      book = Book.new('if tomorrow comes', 'sidney sheldon')
```

Comment:
- [OPTIONAL] I think it would be better if you remove this line because you are already doing this in `before(:each)` ⚒ Don't forget to add the `@` to the book 💪